static/view/home.html

```diff
@@ -6,7 +6,7 @@ <h2 class="header">Web API Confluence</h2>
     </div>
     <div class="card-stacked">
       <div class="card-content">
-        <p>A web UI for visualizing API confluence metrics. These metrics are intended to capture how browser vendors are stretching the elastic band of the web platform.
+        <p>A web UI for visualizing API confluence metrics. These metrics are intended to capture how browser releases are stretching the elastic band of the web platform.
```

Comment:
This parses better as it was, I think? The vendors are the ones performing the action.